test/integration/abuseReportTest.ts

```diff
@@ -20,215 +19,216 @@ const REPORT_NOTICE_REGEXPS = {
 
 
 describe("Test: Reporting abuse", async () => {
-    it('Mjölnir intercepts abuse reports', async function() {
-        this.timeout(60000);
+    for (let endpoint of ['v3', 'r0']) {
```

Comment:
we should probably write why we're doing this because eventually r0 will also be dropped